src/main/java/frc/robot/RobotContainer.java

```diff
@@ -75,7 +78,7 @@ private void configureButtonBindings() {
         
         
         // Instantiate the intake command and bind it 
-        intakeCommand = new OperatorIntakeCommand();
+       // intakeCommand = new OperatorIntakeCommand();
```

Comment:
I know I did this for you, but if it's not being used, you should remove commented out code.